### ISSUE TITLE
How to deploy the backend for this? can some

### ISSUE THREAD

**Issue by yuvaraj119** [open-telemetry/opentelemetry-android#693]
I'm new to opentelemetry and not able to find how to deploy backend for Mobile apps and monitor the Logs, Traces and Metrics for my application. How to deploy backend in my Azure service. I checked Azure don't support Mobile app in open-telemetry.

**Comment by breedx-splk** [comment]
Hey @yuvaraj119. You're question is "How to deploy the backend for this?". Which backend are you referring to?

Backend storage and visualization/observability UIs are both outside the scope of what the OpenTelemetry project provides. Users will typically use of of the many vendors to send their data...Azure is one of them. The data sent will be traces/spans and logs. Pretty sure Azure can handle that, but we can't really give vendor-specific guidance.

If you're just looking to test/evaluate, you can locally run something like [jaeger](https://www.jaegertracing.io/) or the [otel desktop viewer](https://github.com/CtrlSpice/otel-desktop-viewer/tree/main).

**Comment by yuvaraj119** [comment]
How can we deploy opentelemetry collectors for mobile apps in azure cloud.

**Comment by yuvaraj119** [comment]
@breedx-splk Can we use Azure Opentelemetry https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable?tabs=java for Android?

**Comment by breedx-splk** [comment]
@yuvaraj119 Yes, the otel collector (in whatever flavor) should be able to receive OTLP based telemetry from the Android agent here. Installing and configuring up a specific vendor's implementation, however, is mostly outside the scope of what the community here can provide support for.
